hadoop-hdds/docs/content/interface/S3.md

```diff
@@ -93,7 +93,7 @@ If security is not enabled, you can *use* **any** AWS_ACCESS_KEY_ID and AWS_SECR
 
 If security is enabled, you can get the key and the secret with the `ozone s3 getsecret` command (*kerberos based authentication is required).
 
-```
+```bash
 /etc/security/keytabs/testuser.keytab testuser/scm@EXAMPLE.COM
```

Comment:
This should be "kinit -kt /etc/security/keytabs/testuser.keytab testuser/scm@EXAMPLE.COM"